staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -190,7 +192,7 @@ type PodFailurePolicyRule struct {
 	OnPodConditions []PodFailurePolicyOnPodConditionsPattern `json:"onPodConditions" protobuf:"bytes,3,opt,name=onPodConditions"`
 }
 
-// PodFailurePolicy describes how failed pods influence the backoffLimit.
+// podFailurePolicy describes how failed pods influence the backoffLimit.
```

Comment:
types should still start with UpperCase

Reply:
It is mentioned [here](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/#alpha-level) in docs. Should we consider this as a field name or a type?